CHANGELOG.md

```diff
@@ -5,14 +5,13 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 ## [Unreleased](https://github.com/idealista/dotnetcore_role/tree/develop)
 ### Changed
+- *[#1](https://github.com/idealista/dotnetcore_role/issues/1) General role update* @blalop
+
 ### Added
+- *[#1](https://github.com/idealista/dotnetcore_role/issues/1) Debian 11 support* @blalop
 ### Removed
+- *[#1](https://github.com/idealista/dotnetcore_role/issues/1) Debian 9 support* @blalop
```

Comment:
I would add here "BREAKING" :-)

Reply:
Will do when updating the changelog to reflect the new major release :)